resource/js/components/UserList.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import UserPicture from './User/UserPicture';
+
+export default class UserList extends React.Component {
+
+  render() {
+    const users = this.props.users.map((user) => {
+      return (
+        <a data-user-id={user._id} href={'/user/' + user.username} title={user.name}>
+          <UserPicture user={user} size="xs" />
+        </a>
+      );
+    });
+
+    return (
+      <p className="seen-user-list">
```

Comment:
If this `UserList` component is just for `seen-user-list`, the component should be located under `component/SeenUserList/` dir.
As you know, only the root components are allowed to locate direct under `component/` dir.